dracut/29kcrypt/generator.sh

```diff
@@ -9,6 +9,17 @@ GENERATOR_DIR="$2"
 
 oem_label=$(getarg rd.cos.oemlabel=)
 
+// Several things indicate booting from a different media so we should not do anything
+// rd.cos.disable is set on LIVECD and disables mounting of any type
```

Comment:
shouldn't these be `#` or do the double slashes mean something else in shell script?
```suggestion
## Several things indicate booting from a different media so we should not do anything
## rd.cos.disable is set on LIVECD and disables mounting of any type
```

Reply:
🤡 that's me after no knowing to separate golang from bash 🤦‍♂️